src/WebJobs.Script.WebHost/WebHostSettings.cs

```diff
@@ -24,6 +23,8 @@ public class WebHostSettings
 
         public string SecretsPath { get; set; }
 
+        public string TestDataPath { get; set; }
```

Comment:
Can you add a description? What is this for?

Reply:
done